lib/ansible/modules/cloud/vmware/vmware_guest_facts.py

```diff
@@ -209,7 +135,9 @@ def main():
             e = get_exception()
             module.fail_json(msg="Fact gather failed with exception %s" % e)
     else:
-        module.fail_json(msg="Unable to gather facts for non-existing VM %(name)s" % module.params)
+        msg = "Unable to gather facts for non-existing VM"
+        msg += " %s in folder %s" % (module.params['name'], searchpath)
+        module.fail_json(msg=msg)
```

Comment:
This should not be necessary for PEP8. We have a line-length of 160, not the default 80.